src/libPMacc/include/mappings/kernel/BorderMapping.hpp

```diff
@@ -31,10 +31,12 @@ namespace PMacc
 {
 
     /**
-     * This maps onto the border to 1 direction (e.g. TOP, BOTTOM, ...)
+     * This maps onto the border to 1 exchange direction (e.g. TOP, BOTTOM, TOP + LEFT, ...)
      * The area is basically the same as the surrounding guard region
-     * but on the border. Therefore only the "main" directions are allowed
-     * (no diagonal ones)
+     * but on the border. By "adding" directions you limit the area. Examples:
+     * FRONT: Whole top area of the border (Size: ~x*y)
+     * FRONT + LEFT: Only the edge at the intersection of the front and left border (Size: ~y)
+     * FRONT + LEFT + TOP: Only the corner (super)cells (Size: ~1)
```

Comment:
please call it `super cell` without parenthesize, this is a common word for a group of cells and used every in the code and in PIConGPU papers.